app/ide-desktop/common/src/text/english.json

```diff
@@ -480,26 +486,53 @@
   "shareButtonAltText": "Share",
   "userMenuAltText": "User Settings",
 
+  "billedMonthly": "Billed monthly",
+  "billedAnnually": "Billed annually",
+  "tryFree": "Try free for $0 days, then ",
+  "priceTemplate": "$0 per user / month. $1",
   "freePlanName": "Free",
+  "freePlanSubtitle": "Try Enso",
+  "freePlanPricing": "$0 forever",
+  "freePlanFeatures": "Interactive Live Data Analytics; Python, Java, JavaScript Support; Local workflow execution",
+  "freePlanSeatsDescription": "This plan does not allow to adjust seats",
   "soloPlanName": "Solo",
   "soloPlanSubtitle": "For individuals",
-  "soloPlanPricing": "$60 per user / month",
-  "soloPlanFeatures": "10GB Cloud Storage; 1 User; Datalinks; Version Control for files and workflows; Enso Copilot",
+  "soloPlanPricing": "$60 per user / month. Billed annually",
+  "soloPlanFeatures": "Everything from free plan; 10GB Cloud Storage; 1 User; Datalinks; Version Control for files and workflows; Enso Copilot",
+  "soloPlanSeatsDescription": "This plan is only for a single user",
   "teamPlanName": "Team",
   "teamPlanSubtitle": "For small teams",
-  "teamPlanPricing": "$150 per user / month",
-  "teamPlanFeatures": "100GB Cloud Storage; Up to 10 users sharing; Datalinks; Version Control for files and workflows; Enso Copilot; Multiple Lifecycle Environments",
+  "teamPlanPricing": "$150 per user / month. Billed annually",
+  "teamPlanFeatures": "Everything from free plan; 100GB Cloud Storage; Up to 10 users sharing; Datalinks; Version Control for files and workflows; Enso Copilot; Multiple Lifecycle Environments",
+  "teamPlanSeatsDescription": "This plan allows to buy up to $0 seats",
   "enterprisePlanName": "Enterprise",
   "enterprisePlanSubtitle": "For large organizations",
-  "enterprisePlanPricing": "Contact Sales",
-  "enterprisePlanFeatures": "1000+GB Cloud Storage; Unlimited users sharing; Datalinks; Version Control for files and workflows; Enso Copilot; Multiple Lifecycle Environments; Data access and modification logs; Priority Support; Fine-grained files and workflows permissions; Federated Log On",
+  "enterprisePlanPricing": "$250 per user / month. Billed annually",
+  "enterprisePlanFeatures": "Everything from free plan; 1000+GB Cloud Storage; Unlimited users sharing; Datalinks; Version Control for files and workflows; Enso Copilot; Multiple Lifecycle Environments; Data access and modification logs; Priority Support; Fine-grained files and workflows permissions; Federated Log On",
```

Comment:
nit: i know this is existing logic but the strings used in `settingsData` are delimited by `\n`, with the reasoning that `\n` will almost certainly not appear in a bullet point (whereas `;` or `,` may very well appear)

(of course, this is optional)

Reply:
Yeah, makes sense to use the existing logic... on the other hand, when I was creating this string initially(back in March) I thought that for those who might edit this, easier to use `;`, rather than `\n`